packages/buidler-core/test/internal/buidler-evm/stack-traces/test-files/0_6/call-message/external-calls/immutable-variable/c.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.6.5;
+
+contract C {
+  uint immutable x = 10;
+
+  function get2X () public returns (uint) {
+    return 2*x;
```

Comment:
The immutable variable needs to be used for the test to fail. The test passes if the x here is removed. Same comment for the other tests that uses immutable.